firmware/thruna/BMS/src/app/states/app_initState.c

```diff
@@ -45,17 +45,16 @@ static void initStateRunOnTick100Hz(void)
     {
         const bool air_negative_closed = io_airs_isNegativeClosed();
         const bool ts_discharged       = app_tractiveSystem_getVoltage() < TS_DISCHARGED_THRESHOLD_V;
+        const bool missing_hb          = app_heartbeatMonitor_checkFaults();
 
         if (air_negative_closed && ts_discharged)
         {
-            const bool charger_connected         = io_charger_isConnected();
+            const bool charger_connected         = app_canRx_BRUSA_OnStatus_get();
```

Comment:
If the charger requires messages in order to exit it's idle state and send can messages, this won't work as intended. You'd have to send messages to the charger to see if it's alive. Stick to using io_charger_isConnected() until we can test this further